python_modules/dagster/dagster/_serdes/serdes.py

```diff
@@ -829,10 +829,13 @@ def _pack_value(
     if isinstance(val, Enum):
         klass_name = val.__class__.__name__
         if not whitelist_map.has_enum_entry(klass_name):
+            if isinstance(val, str): # serialize StrEnums in their string value
+                return val.value
```

Comment:
@smackesey thoughts on this fallback behavior? The trade-off here is some risk in having in our code a str Enum accidentally does not get whitelisted deserializing as a normal `str` for the upside of allowing users to use str Enums in apis that accept str.